apps/front/src/app/(auth)/register/registerForm/passwordStrength.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useCallback, useEffect, useState } from 'react'
```

Comment:
W tym pliku masz 2 komponenty, style inline, tailwind'a, stylowanie jss od materiala. Wez to moze jakos rozbij i zdecyduj sie na jakies jedno rozwiazanie co do styli